test/test.dart

```diff
@@ -399,9 +399,14 @@ String generateRandomRealmPath() {
 }
 
 final random = Random();
-String generateRandomString(int len) {
-  const chars = 'abcdefghjklmnopqrstuvwxuz';
-  return List.generate(len, (index) => chars[random.nextInt(chars.length)]).join();
+String generateRandomString(int len, {bool allowUnicode = false, String includingSymbols = ""}) {
```

Comment:
This will work but I find the API little weird. 
I would prefer if the symbols passed are replacing the symbols used in the method. So the caller is either using the default implementation of `generateRandomString` or passing a character set specific at the call site. 
Otherwise one needs to really go and check which symbols are the default ones and also most importantly one can not exclude symbols from the default ones. Say for example I want to generate a random string which does not include `abc` in it.

Lets rename `includingSymbols` to characterSet and make it optional and null. If the caller passes it then we use only this characterSet to generate the random string otherwise we use the default one

Reply:
or better yet we initialize the `characterSet` parameter with the default character set right in the method definition

Reply:
If we replace the whole charset with `characterSet `parameter's value then the parameter `allowUnicode` is useless. I will have to add again the method `generateRandomUnicodeString`. I refactored the methods according to your suggestions.